packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/driver.py

```
@@ -0,0 +1,203 @@
+    def wait_instance(self, current: Instance, desired: Optional[Instance]):
+        """
+        Wait for `current` instance to reach the same state as the `desired` instance.
+
+        Desired can also be set to None, which means the instace should not exist.
+        """
+        opts = self.get_timeout_opts()
+        counter = 0
+
+        while True:
+            if counter >= opts['retries']:
+                raise ValueError(f'Instance took too long to be reach the desired state: {instance}')
```

Ruff (0.8.2)

Undefined name `instance`

(F821)